README.md

```diff
@@ -76,13 +76,51 @@ class User
 end
 ```
 
+### Container Version
+
+Instead of extending a class with the `Dry::Initializer::Mixin`, you can include a container with the initializer:
+
+```ruby
+require 'dry-initializer'
+
+class User
+  # notice `{}` syntax for the block, not `do..end`
+  include Dry::Initializer.define {
+    param  :name,  type: String
+    param  :role,  default: proc { 'customer' }
+    option :admin, default: proc { false }
+  }
+end
+```
+
+Now you do not pollute a class with new variables, but isolate them in a special "container" module with the initializer and attribute readers. This method should be preferred when you don't need subclassing.
```

Comment:
Isn't the idea here that we don't pollute the class's _interface_ with the API from dry-initializer?